src/Centreon/Infrastructure/Monitoring/Timeline/TimelineRepositoryRDB.php

```diff
@@ -123,7 +124,7 @@ private function findTimelineEvents(int $hostId, ?int $serviceId): array
         $this->sqlRequestTranslator->setConcordanceArray([
             'type' => 'log.type',
             'content' => 'log.content',
-            'date' => 'log.date',
+            'date' => 'log.date'
```

Comment:
IMHO, it's better to let comma cause a future diff can be better. It avoids to update this line if a new parameter is added in the array

Reply:
Done :100: thanks for the advise